doitlive.py

```diff
@@ -19,12 +18,18 @@
 import getpass
 import socket
 import subprocess
+import click
 from code import InteractiveConsole
 from tempfile import NamedTemporaryFile
 from collections import OrderedDict
-
-import click
-from click import echo, style, secho, getchar
+from termutils import raw_mode
```

Comment:
Follow PEP8's recommendation for import grouping:

```
Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
```

https://www.python.org/dev/peps/pep-0008/#imports